containers/ukftractography/Dockerfile

```diff
@@ -48,17 +48,33 @@ RUN git clone https://github.com/pnlbwh/pnlNipype.git /src/pnlNipype && \
     cd /src/pnlNipype && \
     git checkout e8b9033
 
-# Download UKFTractography
-ENV UKFPATH=/src/ukf
-RUN mkdir -p $UKFPATH && \
-    git clone https://github.com/pnlbwh/ukftractography.git $UKFPATH && \
-    cd $UKFPATH && \
-    cmake && \
-    make \
-    make test
+ENV PYTHONPATH="/src/pnlNipype:$PYTHONPATH"
+
+# Install cmake
+RUN mkdir -p /src/cmake && \
+    curl -sSLO https://github.com/Kitware/CMake/releases/download/v3.21.1/cmake-3.21.1-linux-x86_64.sh && \
+    bash cmake-3.21.1-linux-x86_64.sh --prefix=/src/cmake --skip-license && \
+    rm cmake-3.21.1-linux-x86_64.sh
+
+ENV PATH="/src/cmake/bin:$PATH"
+
+# Use https protocol for git clone
+RUN git config --global url."https://".insteadOf git://
+
+# Install UKFTractography
+RUN git clone https://github.com/pnlbwh/ukftractography.git /src/ukf && \
+    cd /src/ukf && \
+    git checkout 2c859b3 && \
+    mkdir build && \
+    cd build && \
+    cmake ../ && \
+    make
+
+ENV UKFPATH=/src/ukf/build/UKFTractography-build/UKFTractography/bin \
+    PATH="$UKFPATH:$PATH"
 
 # Copy command line script
 COPY run_ukf.sh /src/
 
```

Comment:
We have to make sure we install the dependencies for pnlNipype before we try to run the ukf.py script (for example right now it will fail on not having `psutil`). Going off their [install instructions](https://github.com/pnlbwh/pnlNipype#pnlpipe-software) should suffice!

I can add this as I do my testing, just here for a reminder.

Reply:
Good point! I thought installing the conversion package would take care of all of the dependencies.